entries/p/paycor.com.json

```
@@ -5,7 +5,8 @@
     "tfa": [
       "sms",
       "call",
-      "email"
+      "email",
+      "totp"
     ],
     "documentation": "https://www.paycor.com/security/how-to-keep-your-account-safe",
```

The `totp` method is not mentioned on the old doc page, so that's probably why it was removed. Changing the link to this page should prevent that from happening in the future.
```suggestion
    "documentation": "https://support.paycor.com/s/article/Multi-Factor-Authentication-MFA-Requirements-and-Setup",
```

---

ah, nice catch and thanks for digging up the new URL!